### DISCUSSION TITLE
Is it neccesary to track lazy-lock.json in repo?

### DISCUSSION THREAD

**Discussion by ClSlaid** [ayamir/nvimdots#570] [General]
I think offer a lazy-lock.json.example for backup and example and stop tracking lazy-lock.json is better.

1. users may always want to use the latest packages
2. users may like to keep in a specific version of some package

**Comment by charliie-dev** [comment]
I think that's a good idea!

**Reply by aarnphm** [reply]
I think tracking the lockfile is fine for the main repo. We can have instruction on Wiki how to tell user to untrack the lazy-lock.json

**Comment by ayamir** [comment]
We tracked `lazy-lock.json` to make issues can be reproduced on the level of plugins' version. It should be tracked for this repo. In other words, this repo is incomplete to release if the version control description is missed. Users can fork this repo of course and do anything they want to do.

**Reply by Tgist1024** [reply]
That's true, but if I have personal plugins, I always need to handle the conflict in `lazy-lock.json` when I pull the newest repo.

**Comment by aarnphm** [comment]
Also another thing to note here is that you can use `--assume-unchanged` to the lazy-lock.json
